src/App.js

```diff
@@ -1,21 +1,57 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { Credentials } from './config/Config.js'
+import Recipe from './components/Recipe.js'
+import Error from './components/Error.js'
 
 class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      recipeList: [],
+      errorMessage: []
+    }
+  }
+
+  
+
+  componentDidMount() {
+    const q = 'cakes'
+    const endpoint = `${Credentials.URL}?app_id=${Credentials.APP_ID}&app_key=${Credentials.APP_KEY}&q=${q}`
+    //const endpoint = `${Credentials.URL}?app_id=${Credentials.APP_ID}&app_key=${Credentials.APP_KEY}`
+    
+    const fetchRecipes = () =>
```

Comment:
Consider pulling this function outside of the scope of `componentDidMount`

Something like this
```
 componentDidMount() {
  this.fetchRescipe()
}
fetchRecipes = () => <your fancy code here>
```